packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-page-list.tsx

```diff
@@ -120,19 +121,23 @@ export const AccountPageList: FunctionComponent<AccountPageListProps> = ({
         "data-testid": "create-page-btn",
       }}
     >
-      <TreeView
-        data-testid="pages-tree"
-        tabIndex={-1}
-        sx={{
-          mx: 0.75,
-        }}
-        selected={currentPageEntityId ?? ""}
-        expanded={expanded}
-        onNodeToggle={handleToggle}
-        onNodeSelect={handleSelect}
-      >
-        {formattedData.map((node) => renderTree(node, accountId, 0))}
-      </TreeView>
+      {pagesLoading ? (
+        <PagesLoadingState />
+      ) : (
+        <TreeView
+          data-testid="pages-tree"
+          tabIndex={-1}
+          sx={{
+            mx: 0.75,
```

Comment:
This is a margin duplicated between both branches, but also located fairly far apart – someone changing one may not know they need to change both